Localization.cs

```diff
@@ -0,0 +1,77 @@
+﻿public class Localization
+{
+    public class Phrases
+    {
+        public const string Temperature_OutOfRange = "Temperature_OutOfRange";
+        public const string Temperature_ApproachingMinimum = "Temperature_ApproachingMinimum";
+        public const string Temperature_ApproachingMaximum = "Temperature_ApproachingMaximum";
+
+        public const string Soc_OutOfRange = "Soc_OutOfRange";
+        public const string Soc_ApproachingMinimum = "Soc_ApproachingMinimum";
+        public const string Soc_ApproachingMaximum = "Soc_ApproachingMaximum";
+
+        public const string ChargeRate_OutOfRange = "ChargeRate_OutOfRange";
+        public const string ChargeRate_ApproachingMaximum = "ChargeRate_ApproachingMaximum";
+
+        public const string AllOk = "AllOk";
+    }
+
+    public const string UndefinedError = "Error: Undefined!!!";
+
+    static Language[] languages = new Language[]

```

Comment:
Good idea to keep localization separately :+1: 
Also these should be safeguarded by tests too. Tests are missing...